### DISCUSSION TITLE
question about native zone token

### DISCUSSION THREAD

**Discussion by brentxu** [osmosis-labs/osmosis#30] [Q&A]
what would be the different between staking with the LP token vs the native zone token by itself?  Could this same configuration be for the hub itself?

**Comment by Thunnini** [comment]
You can think of the liquidity pool essentially acting as a 'staking token designer'. The tokens included in the pool and the ratio of the tokens is a way to customize the LP token (and by extension, the staking token) to have specific characteristics that the zone creator desires.

In the current system of proof-of-stake, it only focuses on a zone's sovereignty as only the native tokens are used as measure of voting power. While this isn't necessarily a bad thing, it does mean that the price fluctuation of the zone's staking token means volatility on the safety of the zone itself.

Whereas, in the LP token used as the staking token system, I can essentially peg some/all of the value of my staking token to other tokens.

For example, I can create a zone token with the following characteristics:
* Shares an upside to Cosmos ATOMs price moving up (and the downside as well) --> I can include some ATOMs
* Reduce cost-of-attack exposure to overall cryptoasset volatility --> I can include some DAI
* Have some sovereignty over my zone's voting power --> I can include my zone's utility tokens (not staking tokens as they do not exist. Think photons, rather than ATOMs)

That being said, this also carries significant risk. As the security assumption of proof-of-stake relies on the fact that the cost to acquire a significant portion of the staking token exponentially increases as the attacker attempts to purchase the limited amount of tokens, the multi-token approach of LP staking token means the cost to purchase voting power won't increase exponentially as the LP token is fundamentally diversified. While the 'LP token' staking idea isn't the ultimate solution, it does provide options and customizability where it didn't exist in the 'only native staking token' system.

**Comment by brentxu** [comment]
>the multi-token approach of LP staking token means the cost to purchase voting power won't increase exponentially as the LP token is fundamentally diversified.

Interesting - it sounds like it means that bootstrapping a new network would be difficult, because preexisting holders of Dai, Atom and Utility token could have larger proportions of ownership.

What about testing this idea on Cosmos hubs or zones that are already in existence.  Would this be a heavy lift?
